source/plugins/go/src/extension/extension.go

```diff
@@ -265,13 +265,18 @@ func (e *Extension) GetNamespacesForDataCollection() []string {
 	}
 

```

Comment:
Will we have to change the function where:  getExtensionSettings() is called as well to do something similar? As that is also returning an interface?

Reply:
no because that function already returns map[string]string.